nautobot_golden_config/views.py

```diff
@@ -493,7 +493,7 @@ def get_required_permission(self):
     def plot_visual(aggr):
         """Plot aggregation visual."""
         labels = "Compliant", "Non-Compliant"
-        if aggr["compliants"] is not None:
+        if any((aggr["compliants"], aggr["non_compliants"])):
```

Comment:
Do you mind just adding a comment here to document what the use case this is addressing?

Reply:
Don't mind at all.